src/styles/blocks/block.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  width: $week-size;
+  margin: 1px;
+  padding: 10px;
+
+  display: flex;
+  flex-wrap: wrap;
+
+  @each $day, $size in $days {
+    .calendar--start-day--#{$day}
+    &__block:first-child {
+      margin: $size;
+    }
+  }
+
+  @for $index from 28 to 31 {
+    .calender--month-length--#{$index}
+
+    &__block:nth-child(n + #{$index + 1}) {
+      display: none;
+    }
```

Comment:
Same here, maybe watch the @for and @each tutorial videos again, to help you write them correctly